packages/gateway/src/gateway.js

```diff
@@ -95,6 +81,60 @@ export async function gatewayGet(request, env, ctx) {
   }
 }
 
+/**
+ * Store metrics for winner gateway response
+ *
+ * @param {Request} request
+ * @param {import('./env').Env} env
+ * @param {GatewayResponse} winnerGwResponse
+ */
+async function storeWinnerGwResponse(request, env, winnerGwResponse) {
+  await Promise.all([
+    updateGatewayMetrics(request, env, winnerGwResponse, true),
+    updateGenericMetrics(request, env, winnerGwResponse),
```

Comment:
"Gateway" and "Generic" - same length and begin with G....I thought this was calling the same function both times at first glance. Do you think we should rename? Not sure what would be appropriate - summary/main/overall/universal metrics?

Reply:
Summary is a better naming, I'll change it

Reply:
Probably will also rename the Durable Object. But when we go for production release to reset all version counters